ios/brave-ios/Sources/BraveUI/SwiftUI/LabelDetailView.swift

```diff
@@ -0,0 +1,31 @@
+// Copyright 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+
+public struct LabelDetailView: View {
```

Comment:
1. Move this into AIChat as its not used anywhere else. 
2. iOS 16 already has `LabeledContent` as well so you may want to mark this as obsoleted in iOS 16
3. Include some indication that when detail is nil a progress view is shown (rename the view?)